cypress.config.ts

```diff
@@ -13,6 +13,14 @@ export default defineConfig({
 
       // It's IMPORTANT to return the config object
       // with any changed environment variables
+      // console.log(JSON.stringify(config.browsers))
```

Comment:
remove unnecessary comments (console.log)

Reply:
hard to debug